### PR TITLE
New package: qbe-1.0

### PR DESCRIPTION
**Pull request by costinbadescu**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for these architectures:
  - x86_64-glibc
  - x86_64-musl
  - aarch64-glibc
  - aarch64-musl

### CODE REVIEW

File: srcpkgs/qbe/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'qbe'
+pkgname=qbe
+version=1.0
+revision=1
+archs="x86_64* aarch64*"
```
Comment (Duncaen) [inline srcpkgs/qbe/template:5]: Doesn't it run on all architectures but only can build for those you listed?

Comment (costinbadescu) [reply srcpkgs/qbe/template]: You're right. I updated the PR.

File: srcpkgs/qbe/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'qbe'
+pkgname=qbe
+version=1.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+short_desc="Compiler backend"
```
Comment (ericonr) [inline srcpkgs/qbe/template:7]: This *is* the only short description in the webpage, but I feel like it's too short.... Not sure how to expand it though. Are there any other distros with a QBE package already? Maybe we could take inspiration from them.

**Review by ericonr — COMMENTED** [review]
Tests fail on i686... I think we should consider only running tests on the targets that QBE itself supports (use a case conditional to match on those, then add `make_check=no` for the rest), or adding a dependency on cross toolchains and passing the correct ones to the build system (definitely harder, I don't think it's worth it)

File: srcpkgs/qbe/template (hunk 2 above)
Comment (costinbadescu) [reply srcpkgs/qbe/template:7]: "A simple C compiler backend" is the description of the package on Alpine Linux.

Comment (costinbadescu) [reply srcpkgs/qbe/template:7]: Maybe "Small-footprint compiler backend with full support for the C ABI" or something along those lines?

**Comment by costinbadescu** [comment]
I tested this package further on `x86_64` and `aarch64-musl`. [cproc](https://sr.ht/~mcf/cproc), [harec](https://git.sr.ht/~sircmpwn/harec), and [hare](https://git.sr.ht/~sircmpwn/hare), all of which depend on QBE, compiled successfully with all checks passed.

**Comment by tomekw** [comment]
Is it possible to revive this? I would like to build and maybe even package hare :)

**Comment by BryceVandegrift** [comment]
@tomekw I started a pull request [here](https://github.com/void-linux/void-packages/pull/43951)

**Pull request closed without merging**